inc/wakeonlan.class.php

```diff
@@ -125,7 +125,7 @@ public function prepareRun($taskjobs_id)
                                 $_GET["glpisearchcount2"] = count($_GET['field2']);
                             }
 
-                            $pfSearch = new PluginGlpiinventorySearch();
+                            $pfSearch = new Search();
                             Search::manageGetValues('Computer');
```

Comment:
```suggestion

```
As this function does not exist (even in GLPI 9.5 and FI 9.5) perhaps we can remove it (return statement are never used)

Reply:
Indeed we could, but anyway this class is broken for ages... I'm preparing a PR to remove it entirely, so we can just ignore those issues here.